README.md

```diff
@@ -126,6 +126,31 @@ Name of each event processor is: {even processor name}@{tenant name}
 Access all tenant event processors by retrieving `MultiTenantEventProcessor` only.
 `MultiTenantEventProcessor` acts as a proxy Event Processor that references all tenant event processors.
 
+#### Dead-letter queue
+Only JPA Dead letter queue and In-Memory queues are supported at the moment.
```

Comment:
Unfortunately it will not work, as we depend on JPA mechanism to support multi tenancy wherever we need to store/read something from external database. Thats why its stated that only JPA DLQ is supported, so even if JDBC and Mongo DLQ are available in framework, still will not work in multi-tenancy extension...

Reply:
Hmm gotcha, I recall you sharing this.
However, that would mean this is a statement for the `JpaTokenStore`, `JpaEventStorageEngine`, and `JpaSagaStore` too, right?

Although, setting it up for events is a bit trickier at the moment, this *does not* make it a DLQ-specific argument.
Hence, I think it's better if there's a (sub)topic in the readme explaining why this is the case.

Reply:
I agree, we generally support only JPA out of the box and Axon Server